### PR TITLE
effcient R of group 1

### PR DESCRIPTION
**Pull request by zrc0123**
We add the community contribution file of group 1. Our topic is efficient R.

### CODE REVIEW

File: efficient_r.Rmd [hunk 1]
```diff
@@ -0,0 +1,476 @@
+# Effcient R
+
+Ruochen Zhang, Weipeng Li
+
+```{r setup, include=FALSE}
```
Comment (jontse) [inline efficient_r.Rmd:5]: Please use {r, include=FALSE} instead of {r setup, include=FALSE}

Thanks!

**Review by jontse — CHANGES_REQUESTED** [review]
Please see comments regarding setup.  thanks!

File: efficient_r.Rmd (hunk 1 above)
Comment (zrc0123) [reply efficient_r.Rmd]: I have changed the line into {r, include=FALSE}. Thank you!

Comment (jontse) [reply efficient_r.Rmd]: Thanks!

**Comment by jontse** [comment]
Thank you for your contribution!

**Pull request merged**